ChangeLog

```diff
@@ -2,6 +2,7 @@
 
 * Update password symbol for user creation #262
 * Add support for td-client-ruby 2.x.x #267
+* Add SSL certificate options (--insecure, --ssl-ca-file) for proxy environments
```

Comment:
This section is for current release, please remove. We will add the changelog during release process